java/src/main/java/org/apache/zeppelin/java/JavaInterpreter.java

```diff
@@ -17,19 +17,20 @@
 
 package org.apache.zeppelin.java;
 
+import java.io.File;
```

Comment:
Thanks for the code cleanup!

For the sake of beauty and style. :innocent: 
Would you kindly use google code style for formatting imports.
[settings for IDEA](https://raw.githubusercontent.com/google/styleguide/gh-pages/intellij-java-google-style.xml) [and for others](https://github.com/google/styleguide)

Reply:
Thanks @aka-demik. I've imported the formatting imports file for Eclipse, re-organised imports, saved the file, rebase the PR against master and pushed all of this again. Please let me know what else needs fixing.